src/style.css

```diff
@@ -0,0 +1,29 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  display: inline-block;
+  margin: 2px 6px 2px 2px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  border-radius: 0.5%;
```

Comment:
it is not very clear why you have used percents here. use pixels instead

Reply:
I did not attend when I was checking figma,